docs/resources/gateway_plugin_opentelemetry.md

```diff
@@ -215,5 +215,5 @@ Optional:
 Import is supported using the following syntax:
 
 ```shell
-terraform import konnect_gateway_plugin_opentelemetry.my_konnect_gateway_plugin_opentelemetry "{ \"control_plane_id\": \"9524ec7d-36d9-465d-a8c5-83a3c9390458\",  \"plugin_id\": \"3473c251-5b6c-4f45-b1ff-7ede735a366d\"}"
+terraform import konnect_gateway_plugin_opentelemetry.my_konnect_gateway_plugin_opentelemetry "{ \"control_plane_id\": \"9524ec7d-36d9-465d-a8c5-83a3c9390458\",  \"id\": \"3473c251-5b6c-4f45-b1ff-7ede735a366d\"}"
```

Comment:
@mheap would this be a breaking change for users? 🤔

Reply:
Please ignore - verified that this change fixes the example for import - and works on both v2.4.1 and latest

Reply:
Sorry, this is something I should have put in the description. The reason for the version bump is to fix the import example generation